docs/setup/settings.asciidoc

```diff
@@ -64,10 +66,12 @@ To enable SSL/TLS for outbound connections to {es}, use the `https` protocol
 in this setting.
 
 | `elasticsearch.logQueries:`
```

Comment:
There is another deprecated in this file: server.rewriteBasePath:

Reply:
These statements threw me off:
> In Kibana 7.x, the setting is deprecated. In Kibana 8.0 and later, the default is true.

Does that mean it's deprecated in 7.x but not deprecated in 8.0?